server/frontend/src/components/Products/Products.jsx

```diff
@@ -52,9 +40,8 @@ const MenuProps = {
             let currencies = Array.from(
               new Set(data.products.map((product) => product.currency))
             );
-            console.log(currencies);
-            setCurrencies(currencies);
-            console.log('products',data.products)
+            // setCurrencies(currencies);
+            setCurrencies(["TRY", "USD"]);
```

Comment:
Why did you used setCurrency twice there?

Reply:
for test case kind sir